nameserver/nameserver.go

```diff
@@ -0,0 +1,33 @@
+package nameserver
+
+import (
+	"log"
+	"net"
+)
+
+const dnsMsgSize = 512
+const port = 8853
+
+func getUDPAddr() *net.UDPAddr {
+	return &net.UDPAddr{
+		IP:   net.ParseIP("127.0.0.1"),
+		Port: port,
+	}
+}
+
+// Start will begin listening for DNS queries on port 8853
+func Start() error {
```

Comment:
The mock was being used in tests that we removed, so we removed the mock as well. I explained why we removed those tests in my previous comment.